pkg/kdump/kdump-view.jsx

```diff
@@ -269,18 +282,18 @@ export class KdumpPage extends React.Component {
         const target = this.props.kdumpStatus.target;
         let verifyMessage;
         if (!target.multipleTargets) {
-            let path = target.path || "/var/crash";
+            const path = getLocation(target);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19240-20230824-135223-f118a980-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)